### PR TITLE
FOGL-1718: change "omf" plugin to be named "pi_server"

### PR DESCRIPTION
**Pull request by sstefano69**
develop done
tested.

ready.

NOTE :  
downgrade/upgrade are not managed for this task.

### CODE REVIEW

File: python/foglamp/plugins/north/pi_server/pi_server.py [hunk 1]
```diff
@@ -82,12 +82,12 @@
     "e000000": "general error.",
 }
 # Configuration related to the OMF North
-_CONFIG_CATEGORY_DESCRIPTION = 'OMF North Plugin'
+_CONFIG_CATEGORY_DESCRIPTION = 'PI-Server North Plugin'
 _CONFIG_DEFAULT_OMF = {
     'plugin': {
-        'description': 'OMF North Plugin',
+        'description': 'Server North Plugin',
```
Comment (praveen-garg) [inline python/foglamp/plugins/north/pi_server/pi_server.py]: `PI Server Plugin`

File: python/foglamp/plugins/north/pi_server/pi_server.py [hunk 2]
```diff
@@ -82,22 +82,25 @@
     "e000000": "general error.",
 }
 # Configuration related to the OMF North
-_CONFIG_CATEGORY_DESCRIPTION = 'OMF North Plugin'
+_CONFIG_CATEGORY_DESCRIPTION = 'PI-Server North Plugin'
```
Comment (praveen-garg) [inline python/foglamp/plugins/north/pi_server/pi_server.py]: remove hyphen `PI-Server` > `PI Server`

File: CMakeLists.txt [hunk 3]
```diff
@@ -26,7 +26,7 @@ if (BUILD_C_PLUGINS)
 	if (Modbus)
 		add_subdirectory(C/plugins/south/modbus)
 	endif()
-	add_subdirectory(C/plugins/north/omf)
+	add_subdirectory(C/plugins/north/PI_Server)
```
Comment (praveen-garg) [inline CMakeLists.txt]: I'm not sure about the convention followed here. I would sugegst to name it `PiServer` or `pi-server`

Comment (sstefano69) [reply CMakeLists.txt]: this was asked/defined with Mark.

**Comment by Singhal-Vaibhav** [comment]
@sstefano69 Make sure system tests are considered before we merge this.

File: CMakeLists.txt (hunk 3 above)
Comment (praveen-garg) [reply CMakeLists.txt]: Then its fine.

**Comment by sstefano69** [comment]
@Singhal-Vaibhav , yes I updated/tested the 3 unit tests, they are now using 
```
curl -X POST http://localhost:8081/foglamp/scheduled/task...
```

**Comment by praveen-garg** [comment]
We need to find a way to delete exising omf in python/foglamp/plugins/north/ when run on top of 1.3.1. or existing develop

**Comment by ashish-jabble** [comment]
@sstefano69 Default value for this plugin should be same as defined in `C/plugins/north/PI_Server/CMakeLists.txt` and for this plugin ```\"default\": \"PI_Server\"```

FYI, As @MarkRiddoch has already opened a https://github.com/foglamp/FogLAMP/pull/1011/files to update defaults values for C++ plugins. Also I believe this fix should go into the same PR

**Pull request merged**